src/components/CollectionDetails/index.tsx

```diff
@@ -103,112 +102,38 @@ const CollectionDetails: React.FC<CollectionDetailsProps> = ({
   };
 
   return (
-    <div
-      className="px-4 pt-4 -mx-4 -mt-2 bg-center bg-cover"
-      style={{
-        height: 493,
-        backgroundImage: `linear-gradient(180deg, rgba(17, 24, 39, 0.47) 0%, rgba(17, 24, 39, 1) 100%), url(//image.tmdb.org/t/p/w1920_and_h800_multi_faces/${data.backdropPath})`,
-      }}
-    >
-      <Head>
-        <title>
-          {data.name} - {settings.currentSettings.applicationTitle}
-        </title>
-      </Head>
-      <Transition
-        enter="opacity-0 transition duration-300"
-        enterFrom="opacity-0"
-        enterTo="opacity-100"
-        leave="opacity-100 transition duration-300"
-        leaveFrom="opacity-100"
-        leaveTo="opacity-0"
-        show={requestModal}
+    <>
+      <PageTitle title={data.name} />
```

Comment:
Instead of dropping it here just drop it into the existing JSX so we don't cause every file to be completely indented.